src/c/admin-balance-transfer-item-detail.js

```diff
@@ -0,0 +1,108 @@
+import m from 'mithril';
+import _ from 'underscore';
+import postgrest from 'mithril-postgrest';
+import h from '../h';
+import userVM from '../vms/user-vm';
+import models from '../models';
+import adminUserBalanceTransactionsList from './admin-user-balance-transactions-list';
+
+const adminBalanceTransferItemDetail = {
+    controller(args) {
+        const userBankAccount = m.prop(undefined),
+              metadata = args.item.last_transition_metadata || {transfer_data: {}},
+              transferData = metadata.transfer_data || {},
+              metaBank = transferData.bank_account,
+              userBalance = m.prop({}),
+              transitionBankAccount = m.prop({}),
+              fields = {
+                  admin_notes: m.prop(args.item.admin_notes)
+              },
+              loadingNotes = m.prop(false),
+              submitNotes = () => {
+                  loadingNotes(true);
+                  m.request({
+                      method: 'PUT',
+                      url: `/admin/balance_transfers/${args.item.id}`,
+                      data: {
+                          balance_transfer: {
+                              admin_notes: fields.admin_notes()
+                          }
+                      },
+                      config: h.setCsrfToken
+                  }).then((data) => {
```

Comment:
No need for the data parameter here. Let's try to always add the catch method to our requests also.